powerhome/playbook
# Add Ruby 3.1 & Rails Upgrades for Playbook Website

#### Breaking Changes

Yes. This upgrades Playbook to use `ruby 3.1` which is a major upgrade requiring a major release version. 

#### Runway Ticket URL

https://nitro.powerhrg.com/runway/backlog_items/PLAY-196

#### How to test this

- Ensure all spec tests run (rspec)
- Ensure all kits render in website

#### Checklist:
